src/pages/Signup/Signup.jsx

```diff
@@ -1,11 +1,119 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { validationFunction } from '../../utils/validation';
+import { Link } from 'react-router-dom';
+import { GET_SIGNUP_API } from '../../config';
+import './Signup.scss';
+
+class Signup extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userName: '',
+      userBirth: '',
+      userNum: '',
+      userId: '',
+      userPw: '',
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  signUpSuccess = () => {
+    const { userName, userBirth, userNum, userId, userPw } = this.state;
+    if (this.doValidation() === true) {
```

Comment:
```suggestion
    if (this.doValidation()) {
```
애초에 return 값이 true or false 인데 === true로 검사할 필요 없습니다!

Reply:
수정하였습니다!

Reply:
수정했습니다!